django/db/models/fields/related_descriptors.py

```diff
@@ -151,6 +155,12 @@ def get_prefetch_queryset(self, instances, queryset=None):
                 setattr(rel_obj, rel_obj_cache_name, instance)
         return queryset, rel_obj_attr, instance_attr, True, self.cache_name
 
+    def get_object(self, instance):
+        qs = self.get_queryset(instance=instance)
+        qs = qs.filter(self.field.get_reverse_related_filter(instance))
+        # Assuming the database enforces foreign keys, this won't fail.
+        return qs.get()
```

Comment:
I know this was copied from below but there's no point in not using `get()` directly.

```python
qs = self.get_queryset(instance=instance)
# Assuming the database enforces foreign keys, this won't fail.
return qs.get(self.field.get_reverse_related_filter(instance))
```